README.md

```diff
@@ -1 +1,25 @@
-# merlin
\ No newline at end of file
+# merlin
+
+> Studio Meta WordPress CLI
+
+Command line tool to rapidly create files from templates for WordPress design to work with [studiometa/create-wordpress-project](https://github.com/studiometa/create-wordpress-project).
+
+## Installation
+> **Requires PHP7.3**
+
+Require merlin using [composer](https://getcomposer.org/)
+```
+composer require studiometa/merlin
+```
+## Commands
+**Running commands:**
+```bin
+php merlin command_name
+```
+
+**Available commands:**
+- help
+- list
+- acf:create
```

Comment:
suggestion: Only specify `merlin list` command to prevent to maintain this section up to date each time a command is added to the project